res/ari/resource_channels.c

```diff
@@ -1314,6 +1315,11 @@ static struct ast_channel *ari_channels_handle_originate_with_id(const char *arg
 		return NULL;
 	}
 
+	if (!ast_strlen_zero(args_pre_dial_go_sub)) {
```

Comment:
I think this is too early. Executing here will mean that there will be no access to dialplan variables set in the request itself and connected line will be overwritten if set in the subroutine.